templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Driver</h1>
```

Comment:
The use of `object|yesno:"Update,Create"` might not work as intended because `object` is typically used in detail views, not form views. Consider using a context variable that explicitly indicates whether the form is for creating or updating a driver, and use that variable to determine the heading.